### ISSUE TITLE
Making an Item Enchanted?

### ISSUE THREAD

**Issue by SolidSnake916** [NyaaCat/RPGitems-reloaded#136]
I read all over the tutorial, and tried to print the commands but a lot of commands are cut off in the client for minecraft so I'm unsure... What I want to do it give the sword Sharpness V. and have it have addition powers like Poison on hit from potions which I have... but I cant get rpgitems to say it is enchanted with sharp5, so when it drops off a mob, it would be enchanted and have the power.

**Comment by RecursiveG** [comment]
I'm not very sure what your question is ......
You can use `/enchant` command to enchant rpgitems.
If mobs are equipped with enchanted rpgitems and the drop rate is correctly set. They should drop as normal.

**Comment by SolidSnake916** [comment]
yes I can use /enchant to give me a enchanted item, but the drop mob will not give the item enchanted like I wanted, because rpgitems thinks its not enchanted, so if I did, /rpgitem name give, it gives me the item to what I want yet not enchanted, cause it isn't saved anywhere.

**Comment by SolidSnake916** [comment]
Thanks for fixing the thread to be labeled feature request, i was unaware it was a option in the plugin hence i posting asking if it was possible.

**Comment by RecursiveG** [comment]
To add enchantments onto rpgitems:

1. Take arbitrary item with the desired enchantments in hand
2. Run `/rpgitem [itemName] enchantment clone`

To remove: `/rpgitem [itemName] enchantment clear`
Already enchanted rpgitems won't be reverted, unless clone against items which have no enchantments.

**Comment by SolidSnake916** [comment]
I tested it, it worked nice, but can there be an option to hide Vanilla Enchants since adding an option to move it else where would be annoying, i thought it would be easier for a HIDE option, and then i can add them in Via Lore, like i did in paint real quick to show you how i want to do it. Sorry for re-opening it, but its technically a follow up request, i should had thought of before.

THANKS!!!

Image Preview Below
https://cloud.githubusercontent.com/assets/10108124/16575349/92eab55c-4255-11e6-85ea-7e1cf0ac211a.png
